pkg/storage/engine/testdata/mvcc_histories/target_bytes

```diff
@@ -0,0 +1,58 @@
+## Test opts.TargetBytes.
```

Comment:
The testing seems a little on the light side to me, accepting suggestions on what else to test.